source/base/utilities.cc

```diff
@@ -53,6 +53,7 @@
 #endif
 
 #ifndef DEAL_II_MSVC
+// On Unix-type systems, we use posix-memalign:
```

Comment:
Can you spell that using the correct function name `posix_memalign` so that one can find it when searching for it? (Underscore instead of dash.)